STYLE-GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Exercism's Style Guide
+
+This document acts as a Style Guide for the language and wording used in exercises.
+
+## General principle
+
+All content should be written in US english. 
+In the future other translations may occur, but the "official" Exercism natural langauge is US English.
+
+## Explain or substitude mathmatical and esotoric terms.
+
+In any place that mathematical terms are used they should be explained or substituted out for terms that require less domain knowledge.
+
+Examples:
+- Rather than using "natural numbers", we should use "positive integers" or "zero and positive integers". 
+- If we want to use the phrase "rational numbers", it must be explained in the introduction to the exercise.
```

Comment:
Is a link to an explanation sufficient?

Reply:
I think at the very least, I'd want an example of what is and is not a rational number, and any information that is needed to solve the exercise. For rational numbers, I'd probably have:

> This exercise is about Rational Numbers: Numbers that can be expressed as a fraction of two whole numbers. e.g. 1/4 or 12/6. You can learn more about rational numbers at [maths.com](...).

If I needed to know about signs, I'd add whether rational numbers can be negative or not. If I didn't need to know for the exercise, I wouldn't mention it.

Reply:
Thanks!

Reply:
Feel free to PR that clarification to the guide, etc 🙂